### DISCUSSION TITLE
Using Safari iOS Times Out After 5 Minutes

### DISCUSSION THREAD

**Discussion by Number-3434** [community/community#102595] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The server on mobile appears to be timing out after an extended period of time. When it does this, it will allow you to type things, but not use any green buttons (e.g. comment).

N.B. I am using iOS 16.2 on Safari, and this often times out (i.e. I have to quit Safari and reopen it to get it working again).

I don't know if this issue still persists in later versions of iOS.

**Comment by stevepopovich** [comment]
Hi @Number-3434 are you using the GitHub mobile app or the browser to access GitHub?

**Reply by Number-3434** [reply]
Browser

**Reply by stevepopovich** [reply]
I am going to move this to the correct category, it would be great to provide a video of what you are experiencing!
